router/server.go

```diff
@@ -96,6 +96,15 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router
 		}
 		h.TokenHandler(w, req, provider, false)
 	})))
+	mux.Handle("/api/gettoken", h.ProviderMiddleware(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Do we need to be concerned with protecting this endpoint (i.e. `h.AuthMiddleware(h.SessionInjectorMiddleware())`)?